Venmo/src/main/java/com/braintreepayments/api/VenmoAccountNonce.java

```diff
@@ -65,12 +65,12 @@ static VenmoAccountNonce fromJSON(JSONObject inputJson) throws JSONException {
         return new VenmoAccountNonce(nonce, username, isDefault, json);
     }
 
-    VenmoAccountNonce(String nonce, String username, boolean isDefault, JSONObject json) throws JSONException {
```

Comment:
In this case the `VenmoAccountNonce#fromJSON()` method still uses `has` internally.

We should migrate our codebase to use `opt` methods where it makes sense, but I guess when certain info is missing we would have to return `null` instead of throwing.